<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlowSchedulerDialog</class>
 <widget class="QDialog" name="FlowSchedulerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlowSchedulerDialog</string>
  </property>
  <widget class="QLabel" name="flowOperationsLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operations:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flowSelectOperationButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>=&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flowMoveUpOperationButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="flowPagesLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pages:</string>
   </property>
  </widget>
  <widget class="QListView" name="flowSelectedOperationsListView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>151</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="flowOperationsListView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>121</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="flowSelectedOperationsLable">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Operations:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flowMoveDownOperation">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </widget>
  <widget class="QTreeView" name="flowPagesTreeView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>361</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>399</width>
      <height>359</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="startFlowButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Flow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearDoneButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Done</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>flowSelectOperationButton</sender>
   <signal>clicked()</signal>
   <receiver>FlowSchedulerDialog</receiver>
   <slot>addOperation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startFlowButton</sender>
   <signal>clicked()</signal>
   <receiver>FlowSchedulerDialog</receiver>
   <slot>startFlow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FlowSchedulerDialog</receiver>
   <slot>cancelFlow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearDoneButton</sender>
   <signal>clicked()</signal>
   <receiver>FlowSchedulerDialog</receiver>
   <slot>clearDone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addOperation()</slot>
  <slot>startFlow()</slot>
  <slot>cancelFlow()</slot>
  <slot>clearDone()</slot>
 </slots>
</ui>
